source/common/access_log/access_log_formatter.cc

```diff
@@ -46,6 +46,22 @@ StreamInfoFormatter::FieldExtractor sslConnectionInfoStringExtractor(
   };
 }
 
+StreamInfoFormatter::FieldExtractor upstreamSslConnectionInfoStringExtractor(
```

Comment:
Drive by: given the size of this change can we please split this part out into a follow up? Let's just do the SSL info shared pointer change in a single PR?

Reply:
Yes, I only needed access log change to validate the build, can take it away from the PR.